app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,15 @@
+class AdminController < ApplicationController
+
+  def index
+    unless current_user_is_admin?
+      flash[:error] = "You do not have sufficient permissions to view this page"
```

Comment:
Okay, probably something to figure out locally, then (at a later date).  I think we had issues with the HydraController in the past pulling in way too much stuff.